pyramid_swagger/exceptions.py

```diff
@@ -2,10 +2,10 @@
 import sys
 
 import six
-from pyramid.httpexceptions import HTTPClientError, HTTPInternalServerError
+from pyramid.httpexceptions import HTTPBadRequest, HTTPInternalServerError
 
 
-class RequestValidationError(HTTPClientError):
+class RequestValidationError(HTTPBadRequest):
```

Comment:
Yeah, `HTTPClientError` doesn't have a status code so it'll default to `520`.  The behavior you were relying on prior to pyramid 1.6 still exists but inheriting from `HTTPBadRequest`.  You could also just set the status codes manually if you prefer that.